quiz1/Circle.java

```diff
@@ -0,0 +1,29 @@
+package quiz1;
+
+public class Circle {
+    Circle(double radius) {
+        this.radius = radius;
+    }
+
+    private final double radius;
```

Comment:
This is a minor point but it's more conventional to define constants, instance variables before constructors in Java.

Reply:
Ahhh I see. Thanks!